test/passes/fannkuch3.bin.txt

```diff
@@ -2534,7 +2534,7 @@ Abbrev table for offset: 0x00000000
 
 0x00000082:   DW_TAG_subprogram [10] *
                 DW_AT_low_pc [DW_FORM_addr]	(0x0000000000000006)
-                DW_AT_high_pc [DW_FORM_data4]	(0x00000397)
+                DW_AT_high_pc [DW_FORM_data4]	(0x00000383)
```

Comment:
I guess this means that Binaryen shrank the function? Is that expected for this test?

Reply:
Yes, it shrinks it a little even without running optimizations, due to LEBs being compressed (which clang doesn't do in all places). I verified that size is correct.